pkg/pss/evaluate.go

```diff
@@ -61,7 +69,7 @@ func evaluatePSS(level *api.LevelVersion, pod corev1.Pod) (results []pssutils.PS
 	return results
 }
 
-func exemptKyvernoExclusion(defaultCheckResults, excludeCheckResults []pssutils.PSSCheckResult, exclude kyvernov1.PodSecurityStandard) []pssutils.PSSCheckResult {
+func exemptPodLevelExclusion(defaultCheckResults, excludeCheckResults []pssutils.PSSCheckResult, exclude kyvernov1.PodSecurityStandard, pod *corev1.Pod, matching *corev1.Pod) []pssutils.PSSCheckResult {
```

Comment:
I think there's actually a lot of duplicate logic between `exemptPodLevelExclusion` and `exemptContainerLevelExclusion`, so maybe we can pump that duplicate logic into single functions?